test/services/query-builder.test.js

```diff
@@ -31,7 +31,9 @@ describe('services > query-builder', () => {
     describe('with a hasOne relationship', () => {
       function setup() {
         const target = {
+          primaryKeyAttributes: ['pid'],
```

Comment:
So you changed the primary key from `id` to `pid` in this test, right?
Is that really necessary? `id` as a primary key is simpler to understand.